test/index.test-d.ts

```diff
@@ -21,6 +21,15 @@ inject(dispatch, { method: 'get', url: '/' }, (err, res) => {
   expectType<Function>(res.json)
 })
 
-expectType<Promise<Response>>(inject(dispatch, { method: 'get', url: '/' }))
+inject(dispatch)
+  .get('/')
+  .end((err, res) => {
+    expectType<Error>(err)
```

Comment:
Here you can use `expectError` :)

Reply:
I just followed L18 to keep the coding style consistent. If using `expectError` is better, I suggest to replace these all.